arduino-ide-extension/src/browser/style/index.css

```diff
@@ -19,6 +19,13 @@
 @import './custom-codicon.css';
 @import './progress-bar.css';
 
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 300;
+  src: url(https://fonts.gstatic.com/s/opensans/v29/memSYaGs126MiZpBA-UvWbX2vVnXBbObj2OVZyOOSr4dVJWUgsjZ0B4gaVI.woff2) format('woff2');
```

Comment:
It would be better to have it at build time instead of fetching it. Or how does this work? 😊 Thank you!

Reply:
I added it taking into account how something like this is already implemented here:
https://github.com/arduino/arduino-ide/blob/7f2b849963c47efd20aba4329be154c8ee4cf855/arduino-ide-extension/src/browser/style/fonts.css#L1-L10
I'll investigate how to add it at compile time. Thank you.

Reply:
@francescospissu I think Akos comment was related to the fact that 
`src: url(https://fonts.gstatic.com/s/opensans/v29/memSYaGs126MiZpBA-UvWbX2vVnXBbObj2OVZyOOSr4dVJWUgsjZ0B4gaVI.woff2) format('woff2');
` will not work without internet connection. (see the HTTP fetch)

while `url('fonts/FontAwesome.woff?nuchcq') format('woff') ` does, as the font is in the source files.

I agree we should avoid the fetch if possible